extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -38,13 +40,50 @@ export const buildOpenAttachmentElement = (element) =>
     </a>`;
 
 /**
- * Renders the open attachment UI affordance.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+export const buildOpenInlineAttachmentElement = (element) =>
+  htmlFor(element)`
+    <a class="
+        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset"
+        role="button">
+      <div class="i-amphtml-story-inline-page-attachment-chip">
+        <div class="i-amphtml-story-inline-page-attachment-img"></div>
+        <div class="i-amphtml-story-inline-page-attachment-img-2"></div>
```

Comment:
a11y:
These need to be visible to screen readers.
This can be achieved by adding `role="image"`
They also an `alt` attribute. `alt` can be empty by default and `cta-image-alt` or `cta-img-2-alt` can be propagated to it.

Reply:
Update, it's `aria-label` not, `alt`. My mistake :) 
[role-img docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img)